include/experimental/__p0009_bits/layout_left.hpp

```diff
@@ -132,6 +132,7 @@ class layout_left::mapping {
       )
     )
     MDSPAN_CONDITIONAL_EXPLICIT((!std::is_convertible_v<typename _Mapping::extents_type, extents_type>))
+    MDSPAN_INLINE_FUNCTION MDSPAN_IMPL_CONSTEXPR_14
```

Comment:
I would say `constexpr` considering that this is guarded with `#if MDSPAN_HAS_CXX_17`.
Also, I am not expecting we will fix here, but I thought we had dropped support for C++14.
@nmm0 ?

Reply:
> I am not expecting we will fix here

What do you mean ?

My main concern was about the device annotation actually, I got warnings when compiling DDC. Is adding `MDSPAN_INLINE_FUNCTION` ok ?

Reply:
I am saying I would want
```suggestion
    MDSPAN_INLINE_FUNCTION constexpr
```

Reply:
Fine with me

Reply:
(both places)